Update 3-installation.md (#178)
nativescript-vue/nativescript-vue.org: content/docs/en/getting-started/3-installation.md

Verify the installation was successful by running `tns` in your terminal. You should see a list of the available commands.

### Quick Installation

> Choose a [template](https://nativescript-vue.org/docs/getting-started/templates/) 

Reviewer: ```suggestion
Create a new project from one of the available [templates](https://nativescript-vue.org/docs/getting-started/templates/). 
```

Consider providing the `tns create` line as part of the commands below as well.

Reviewer 2: The templates page no longer exists, so this part is obsolete.

```shell
$ cd myproject
$ npm i
$ tns setup
```

> This will install all ios and android requirements rather than having to manually install them all.

Reviewer: ```suggestion
This installs all iOS and Android dependencies for you instead of you having to manually install them all.
```
